src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/mega/GregTechMetaTileEntity_MegaAlloyBlastSmelter.java

```diff
@@ -363,9 +363,7 @@ protected boolean processRecipe(ItemStack[] tItems, FluidStack[] tFluids) {
         if (tItems.length <= 0 && tFluids.length <= 0) return false;
         long tVoltage = this.getMaxInputVoltage()
                 / this.getExoticAndNormalEnergyHatchList().size();
-        long tAmps = (long) (GT_ExoticEnergyInputHelper.getMaxInputAmpsMulti(this.getExoticEnergyHatches()) * 0.8D
-                + GT_ExoticEnergyInputHelper.getMaxInputAmpsMulti(
-                        this.mEnergyHatches)); // Use 80% of the available amps for exotic hatches to get working amps
+        long tAmps = GT_ExoticEnergyInputHelper.getMaxWorkingInputAmpsMulti(getExoticAndNormalEnergyHatchList());
```

Comment:
why not use getMaxInputAmps instead of this? After all MABS extends the ExndedPowerMultiblockbase